### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rajula96reddy**
Solves #2

### CODE REVIEW

**Comment by chris-short** [comment]
@rajula96reddy Any idea as to why that test is failing?

**Comment by kaslin** [comment]
@chris-short looks like it's failing for exactly the thing I was about to mention:
Run gr2m/merge-schedule-action@v1.x
Handling pull request edited for https://github.com/kubernetes-sigs/contributor-tweets/pull/5
##[error]Setting a scheduled merge is not allowed from forks

If scheduling a tweet is not allowed from forks, and I think we can only take PRs from forks(?), then we are currently unable to schedule tweets, right? Until such a time as scheduling from forks is supported?

Aside from that, it looks good as a first pass on the readme.

Also I believe my permissions issue is fixed so we should be able to assign me now.
/assign @kaslin

**Comment by rajula96reddy** [comment]
@chris-short @kaslin I was about to bring this issue to your notice. Like Kaslin mentioned, 'merge' is not possible from forked PR's as of now (https://github.com/gr2m/merge-schedule-action/pull/6). So that leaves us to remove the 'schedule merge' Github action for now and look out for other options. We won't run into failed checks if we remove this action.

The 'schedule merge' action basically merges the PR automatically at the given time on a given day. Given this, we can explore if we can schedule a merge using Prow or maybe write our own custom action if needed.

File: GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Guidelines
+
+### File Name
+Make sure the `.tweet` file name created follows the convention `dd-mm-yyyy-<tweet-type>.tweet`
+
+### File Contents
+Make sure the number of characters are under a tweet limit. You can add emojis to the text as well.
```
Comment (gkarthiks) [inline GUIDELINES.md:7]: we could also add the suggestion like below?

while doing the "@" mention of some one, please make sure it is correct, as the bot currently doesn't validate the tagging

Comment (rajula96reddy) [reply GUIDELINES.md:7]: Yes. Good catch. I will add this as well.

**Pull request merged**